docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+      selectedLabel: 'Select an item'
+    }
+  },
+  methods: {
+    clickHandler (msg, val, label) {
+      if (val !== undefined) {
+        this.selectedItem = val
+      }
+
+      if (label) {
+        this.selectedLabel = label
+      }
+
+      this.$toaster.open(msg)
+    }
+  }
+}
+</script>
+```
+
+### showCaret
+
+Use this prop if you would like the trigger button to display the caret.
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" show-caret />
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="items"
+  show-caret
+/>
+```
+
+### kpopAttributes
+
+Use the `kpopAttributes` prop to configure the **KPop** [props](/components/popover.html) dropdown.
+
+<div>
+  <KCard
+    title="Card Title"
+    body="Body Content"
+  >
+    <template #actions>
+      <KDropdownMenu
+        :kpop-attributes="{
+          popoverClasses: 'mt-5',
+          width: '180'
+        }"
+        :items="deepClone(defaultItemsUnselected)"
+      >
+        <template #default>
+          <KButton
+            appearance="secondary"
+            size="small"
+            class="non-visual-button"
+          >
+            <template #icon>
+              <KIcon
+                icon="more"
+                color="var(--black-400)"
+                size="16"
+              />
+            </template>
+          </KButton>
+        </template>
+      </KDropdownMenu>
+    </template>
+  </KCard>
+</div>
+
+```html
+<KCard
+  title="Card Title"
+  body="Body Content"
+>
+  <template #actions>
+    <KDropdownMenu
+      :kpop-attributes="{
+        popoverClasses: 'mt-5',
+        width: '180'
+      }"
+      :items="deepClone(defaultItemsUnselected)"
+    >
+      <template #default>
+        <KButton
+          appearance="secondary"
+          size="small"
+          class="non-visual-button"
+        >
+          <template #icon>
+            <KIcon
+              icon="more"
+              color="var(--black-400)"
+              size="16"
+            />
+          </template>
+        </KButton>
+      </template>
+    </KDropdownMenu>
+  </template>
+</KCard>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with `disabledTooltip` prop.
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<div>
+  <KDropdownMenu
+    label="Documentation"
+    :disabled="true"
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItemsUnselected)"
+  />
+</div>
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :disabled="true"
+  disabled-tooltip="You can't click me"
+  :items="deepClone(defaultItemsUnselected)"
+/>
+```
+
+## Slots
+
+There are 2 supported slots:
+
+- `default` - The trigger element for opening/closing the menu. Returns the `isOpen` state.
+- `items` - For an example of using the items slot see the [`KDropdownItem`](#KDropdownItem) section.
+
+<div>
+  <KDropdownMenu :items="deepClone(defaultItemsUnselected)">
```

Comment:
Similar to the changes we made on some other components, can we update the examples to not link away from the current page (eg linking to the button docs)? Maybe just show a tooltip or something instead?

Reply:
So, in order to have actionable items using the `items` prop, they have to be links. I compromised by only linking to anchor tags within the current page.